release/ray_release/scripts/ray_bisect.py

```diff
@@ -118,22 +136,26 @@ def _sanity_check(
         f"Sanity check passing revision: {passing_revision}"
         f" and failing revision: {failing_revision}"
     )
-    outcomes = _run_test(test, [passing_revision, failing_revision], run_per_commit)
+    outcomes = _run_test(
+        test, [passing_revision, failing_revision], run_per_commit, is_full_test
+    )
     if any(map(lambda x: x != "passed", outcomes[passing_revision].values())):
         return False
     return any(map(lambda x: x != "passed", outcomes[failing_revision].values()))
 
 
 def _run_test(
-    test: Test, commits: Set[str], run_per_commit: int
+    test: Test, commits: Set[str], run_per_commit: int, is_full_test: bool
```

Comment:
maybe add a default value for `is_full_test`? so that you do not need to change most of the existing unit tests?

Reply:
totally!